docs/src/pages/docs/functions/helpers.md

```diff
@@ -743,7 +743,7 @@ yourself constantly concatenating the result of this function into another
 `crocks/helpers/omit`
 
 ```haskell
-omit :: Foldable f => f String -> Object -> Object
+omit :: Foldable f => f ([ String ] | List String) -> Object -> Object
```

Comment:
We should remove the `Foldable f => f` portion. At one time that working or was going to work with any foldable structure. Also `List` should be removed as this function only applies to Array at the moment. `List` does not have the required `indexOf`